message_parser/src/test/java/cumulus_message_adapter/message_parser/MessageParserTest.java

```diff
@@ -104,6 +104,49 @@ public void testLoadAndUpdateRemoteEvent() {
         }
     }
 
+    @Test
+    public void testLoadAndUpdateRemoteEvent_s6Success() {
+        MessageAdapter messageAdapter = new MessageAdapter();
+        //"Run non-Hanging Test"
+        try {
+            String inputJsonString = AdapterUtilities.loadResourceToString("s6.input.json");
+            // the message is not changed
+            String expectedJsonString = AdapterUtilities.loadResourceToString("basic.input.json");
```

Comment:
I think you can strip out all of the parts of this dealing with expectations since we don't test against them.